driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -81,4 +100,27 @@ public static ArrayExpression<BooleanExpression> ofBooleanArray(final boolean...
         return new MqlExpression<>((cr) -> new BsonArray(result));
     }
 
+
+    public static ArrayExpression<IntegerExpression> ofIntegerArray(final int... ofIntegerArray) {
```

Comment:
Should we have a separate PR for all the `of` methods instead of sprinkling them in like this?

Reply:
I introduce and test the literal `of` methods for each expression type in the corresponding PR; I don't think they should all be introduced separately. For example, the boolean `of` was part of the prior boolean PR. But I had to advance these into this PR to properly test eq/lt/etc.